mappings/net/minecraft/util/registry/DynamicRegistryManager.mapping

```diff
@@ -14,6 +14,7 @@ CLASS net/minecraft/class_5455 net/minecraft/util/registry/DynamicRegistryManage
 	FIELD field_26733 BUILTIN Lnet/minecraft/class_5455$class_5457;
 	METHOD method_30519 load (Lnet/minecraft/class_5455$class_5457;Lnet/minecraft/class_5382;)V
 		COMMENT Loads a dynamic registry manager from the resource manager's data files.
+		ARG 0 registryManager
```

Comment:
Hmm, this method looks a bit weird. worth revisiting in another pr as the doc I wrote seems outdated